rebasehelper/application.py

```diff
@@ -194,6 +195,9 @@ def _prepare_spec_objects(self):
                 raise RebaseHelperError('Could not determine latest upstream version '
                                         'and no SOURCES argument specified!')
 
+        if parse_version(self.conf.sources) <= parse_version(self.spec_file.get_version()):
+            raise RebaseHelperError("Your local version is the latest available version.")
```

Comment:
You shouldn't expect that the version is the lastest available, it could have been specified on the command line and could be some older version. So I think better message is this: `Current version is equal to or newer than the requested version, nothing to do.`.

Reply:
You should also modify the commit message accordingly.